plugins/aea-ledger-ethereum-hwi/setup.py

```diff
@@ -43,7 +43,7 @@
         "ipfshttpclient==0.8.0a2",
         "eth-account==0.5.6",
         "open-aea-ledger-ethereum~=1.31.0",
-        "apduboy>=0.5.0",
+        "apduboy @ git+https://github.com/LedgerHQ/apduboy.git",
```

Comment:
@angrybayblade Where did you find this lib? 
pretty cool, but not a lot of maintainers